presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestSpatialJoinPlanning.java

```diff
@@ -87,6 +87,26 @@ public void testSpatialJoinIntersects()
                                 anyTree(project(ImmutableMap.of("geometry_b", expression("ST_GeometryFromText(cast(wkt_b as varchar))")), anyTree(values(ImmutableMap.of("wkt_b", 0))))))));
     }
 
+    @Test
+    public void testDistanceQuery()
```

Comment:
Another case when distance is an expression

Reply:
@arhimondr There are 2 cases already. One with fixed radius and one with `300 / (111321 * cos(radians(b.lat)))`. Is there some other case you are thinking about?

Reply:
Sorry, overlooked